usdrate/storage/postgres/postgres_test.go

```diff
@@ -0,0 +1,51 @@
+package postgres
+
+import (
+	"testing"
+	"time"
+
+	_ "github.com/lib/pq" // sql driver name: "postgres"
```

Comment:
I don't think we need to import this cuz we have it in `testutil.MustNewDevelopmentDB()`